test/fuzzing/run-draw-fuzzer-tests.py

```diff
@@ -1,66 +1,53 @@
 #!/usr/bin/env python3
 
-import sys, os, subprocess, tempfile, shutil
+import sys
+import os
+from hb_fuzzer_tools import (
+    run_command,
+    chunkify,
+    find_fuzzer_binary,
+    gather_files
+)
 
+def main():
+    srcdir = os.getenv("srcdir", ".")
+    EXEEXT = os.getenv("EXEEXT", "")
```

Comment:
I think this is a leftover from autotools build. We don’t seem to set this envvar anywhere.